website/templates/project/addons.mako

```diff
@@ -124,32 +125,35 @@
                                         <span aria-hidden="true">&times;</span>
                                     </button>
                                     <div>
-                                        <h4>Don’t see your GitHub Organization repositories?</h4>
+                                        <h4>Don’t see your GitHub organization repositories?</h4>
                                         <p>
                                             You may need to reauthorize your GitHub access token.
                                             Follow the steps in the <a class="alert-link" href="http://help.osf.io/a/850865-reauthorize-github" target="_black">help guide</a> to resolve the issue. <br>
                                         </p>
                                         <p>
-                                            Please contact support <a class="alert-link" href="mailto:support@osf.io">support@osf.io</a> if you have questions.
+                                            Please contact <a class="alert-link" href="mailto:support@osf.io">support@osf.io</a> if you have questions.
                                         </p>
                                     </div>
                                 </div>
+                            </div>
                         % endif
                         % if addon['addon_short_name'] == 'zotero':
-                            <div id='zotero-group-library-alert' class='scripted dismissible-alerts'>
```

Comment:
It's generally considered bad practice to break one thing for the sake of fixing another. I've made a change (c0902610e09effe51d14dce30121a7e4014dc9fe) that allows for multiple `dismissible-alerts` to work on the same page, and added this back in. The short version is that because we were using CSS classes (not expected to be unique) like DOM id's (expected to be unique), things were breaking.